code/modules/clothing/glasses/_glasses.dm

```diff
@@ -211,6 +212,12 @@
 	. = ..()
 	REMOVE_TRAIT(user, TRAIT_BOOZE_SLIDER, CLOTHING_TRAIT)
 
+/obj/item/clothing/glasses/sunglasses/chemical
+	name = "science glasses"
+	icon_state = "sunhudsci"
+	desc = "A pair of tacky purple sunglasses that allow the wearer to recognize various chemical compounds with only a glance."

```

Comment:
Thanks guys for your ideas

Reply:
should be science HUDSunglasses

Reply:
should go in this file
https://github.com/tgstation/tgstation/blob/master/code/modules/clothing/glasses/hud.dm

Reply:
As long as you dont name them **scunglasses** i will not rest

Reply:
#scun# as in meth?

Reply:
Science + sunglasses = scunglasses

Reply:
maybe in another life or PR